src/topp/OpenSwathAssayGenerator.cpp

```diff
@@ -117,6 +117,7 @@ class TOPPOpenSwathAssayGenerator :
     registerFlag_("enable_detection_unspecific_losses", "set this flag if unspecific neutral losses (H2O1, H3N1, C1H2N2, C1H2N1O1) for detection fragment ions should be allowed");
     registerFlag_("enable_identification_specific_losses", "set this flag if specific neutral losses for identification fragment ions should be allowed");
     registerFlag_("enable_identification_unspecific_losses", "set this flag if unspecific neutral losses (H2O1, H3N1, C1H2N2, C1H2N1O1) for identification fragment ions should be allowed");
+    registerFlag_("enable_identification_ms2_precursors", "set this flag if MS2-level precursor ions for identification should be allowed");
```

Comment:
maybe explain a bit more what MS2 precursors are and how they help, e.g. "enable extraction of the precursor signal from the fragment ion data (MS2-level). This may help in identification if the MS1 signal is weak" ?